src/types/IDPEntityDescriptor.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Do not edit this file it is auto-generated by italia-utils / gen-api-models.
+ * See https://github.com/teamdigitale/italia-utils
+ */
+/* tslint:disable */
+
+import * as t from "io-ts";
+
+// required attributes
+const IDPEntityDescriptorR = t.interface({
+  cert: t.readonlyArray(t.string, "array of string"),
```

Comment:
you can use [NonEmptyString](https://github.com/teamdigitale/io-ts-commons/blob/master/src/strings.ts#L13) from `italia-ts-commons` that also checks that the string is not empty

Reply:
fixed on commit baa27f1